### PR TITLE
OCPBUGS-2306: fix number spinner input

### PR DESCRIPTION
**Pull request by sahil143**
**Fixes**: https://issues.redhat.com/browse/OCPBUGS-2306

**Analysis / Root cause**: 

User not able to input value in NumberSpinner directly

**Solution Description**: 

add a custom onchange callback for the NumberSpinner formik field

**Screen shots / Gifs for design review**: 

Kapture 2022-10-27 at 12 06 42

**Unit test coverage report**: 

**Test setup:**

**Browser conformance**: 

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: frontend/public/components/utils/number-spinner.tsx [hunk 1]
```diff
@@ -22,7 +22,9 @@ export const NumberSpinner: React.FC<NumberSpinnerProps> = ({
         inputProps={{ ...inputProps }}
         className={className}
         minusBtnAriaLabel={t('public~Decrement')}
+        minusBtnProps={{ 'data-test-id': 'Decrement' } as ButtonProps}
```
Comment (invincibleJai) [inline frontend/public/components/utils/number-spinner.tsx:25]:
```suggestion
        minusBtnProps={{ 'data-test': 'Decrement' } as ButtonProps}
```

Comment (sahil143) [reply frontend/public/components/utils/number-spinner.tsx:25]: In most cased we use `data-test-id` and also for integration tests.

**Pull request merged**